bolt/test/X86/dynamic-relocs-on-entry.s

```diff
@@ -0,0 +1,25 @@
+// This test examines whether BOLT can correctly update when
```

Comment:
In order to check that BOLT can correctly ~update~  process the binary please add extra check that relocation new symbol address is correct in output binary